.gitignore

```diff
@@ -1,2 +1,23 @@
 .DS_Store
 /target
+
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
```

Comment:
We're creating an executable, so Cargo.lock should *not* be in .gitignore. Please remove.

Reply:
Done.